src/main/java/dk/kb/image/PalettePicker.java

```diff
@@ -0,0 +1,100 @@
+package dk.kb.image;
+
+import java.awt.Color;
+
+import com.badlogic.gdx.utils.FloatArray;
+import com.github.tommyettinger.colorful.oklab.Palette;
+
+public class PalettePicker {
+
+    /**
+     * Defines simple color buckets. 
+     * Used in getMostUsedRGBColor() to extract primary color.
+     * @return an integer array of RGB colors.
+     */
+    public static int[] defineSimpleBuckets(){
+         // Define colors as integers in array
+        return new int[]{
+            Color.RED.getRGB(),
+            Color.GREEN.getRGB(),
+            Color.BLUE.getRGB(),
+            Color.YELLOW.getRGB(),
+            Color.CYAN.getRGB(),
+            Color.MAGENTA.getRGB()
+            };
+    }
+
+    /**
+     * Create a float array of 256 colors in the OKlab colorspace
+     * This color palette is called the Aurora Palette and has originally been created by DawnBringer. 
+     * The palette contains 256 colors chosen for game development.
+     * @return a primitive float array with 256 colors in the OKlab colorspace
+     */
+    public static float[] arbitraryOKlabBuckets(){
+        FloatArray array = Palette.LIST;
+        float[] buckets = array.toArray();
+        return buckets;
+    }
+    
+    /**
+     * Convert a string[] of hex colors to a float[] of colors in the OKlab Colorspace.
+     * @return a float[] of OKlab colors.
+     */
+    public static float[] smkOkLabBuckets(){
+        float[] smkOkLabBuckets = ColorConversion.convertHexArrayToOKlabArray(smkHexPalette());
+        return smkOkLabBuckets;
+    }
+
+    public static int[] smkRgbBuckets(){
+        int[] smgRgbBuckets = new int[PalettePicker.smkHexPalette().length];
+        String[] hexPalette = PalettePicker.smkHexPalette();
+        for (int i=0; i<smgRgbBuckets.length; i++){
+            Color color = Color.decode(hexPalette[i]);
+            smgRgbBuckets[i] = color.getRGB();
+        }
+
+        return smgRgbBuckets;
+    }
+    /**
+     * Create a string[] of 256 hex colors used by SMK (National Gallery of Denmark).
+     * It is not clear how this palette has been constructed in the first place.
+     * @return a string[] of 256 hex values.
+     */
+    public static String[] smkHexPalette(){
+        // 256 color palette from SMK Open (National Gallery of Denmark).
+        // Used in their image discovery at: https://open.smk.dk/art/advanced/colors?q=*&page=0 
+        // It is unclear how they have come to use this exact colorpalette 
+        return new String[]{
```

Comment:
Minor style note: Instead of having a method that always return the same array, we can define a constant:
```
public static String[] SMK_HEX_PALETTE = new String(){
"#FDDCE5", "#FADCEC", ...
```